api/v1alpha1/bundle_types.go

```diff
@@ -93,6 +96,13 @@ type LocalSource struct {
 	ConfigMapRef *ConfigMapRef `json:"configMap"`
 }
 
+type HTTPSource struct {
```

Comment:
+1 To Dan's comment. Let's make sure to update all references to HTTP to be HTTPS. I can't imagine a world where we would want to support HTTP.